### PR TITLE
Added Bolivian providers

### PR DESCRIPTION
**Pull request by josegus**
Added Bolivian providers for Person, PhoneNumber and Address

### CODE REVIEW

File: src/Faker/Provider/es_BO/Person.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}'
+    );
+
+    protected static $firstNameMale = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/es_BO/Person.php]: Please mention a valid and MIT compatible datasource in the phpDoc

File: src/Faker/Provider/es_BO/Person.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        'Samuel', 'Santiago', 'Santino', 'Sebastián', 'Sergio', 'Simón',
+        'Thiago', 'Tomas',
+        'Valentino', 'Valentín', 'Vicente', 'Victor',
+        'Walter', 'Willian', 'Williams', 'Wilson',
+        'Xavier',
+        'Yeison',
+        'Zenon'
+    );
+
+    protected static $firstNameFemale = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/es_BO/Person.php]: same

File: src/Faker/Provider/es_BO/Person.php [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        'Rafaela', 'Rebeca', 'Regina', 'Renata', 'Romina', 'Rosario',
+        'Salomé', 'Samantha', 'Sara', 'Silvana', 'Silvina', 'Sofía', 'Sophie', 'Sonia',
+        'Tatiana', 'Thais',
+        'Valentina', 'Valeria', 'Valery', 'Victoria', 'Violeta',
+        'Ximena',
+        'Yessica',
+        'Zoe'
+    );
+
+    protected static $lastName = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/es_BO/Person.php]: same

File: src/Faker/Provider/es_BO/Person.php [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+    /**
+     * Generate a Cedula de Identidad (CI) number
+     *
+     * Bolivians commonly use only 7 digits.
+     *
+     * @example '6987452'
+     *
+     */
+    public function cedulaIdentidad()
```
Comment (fzaninotto) [inline src/Faker/Provider/es_BO/Person.php]: please mention this formatter in the locale specific section of the README

File: src/Faker/Provider/es_BO/Person.php (hunk 1 above)
Comment (josegus) [reply src/Faker/Provider/es_BO/Person.php]: Sorry, what is a valid mit datasource? I'm new in this. I've seen other providers and all those looks like mine.

Comment (josegus) [reply src/Faker/Provider/es_BO/Person.php]: Done

File: src/Faker/Provider/es_BO/Person.php (hunk 2 above)
Comment (josegus) [reply src/Faker/Provider/es_BO/Person.php]: Done

File: src/Faker/Provider/es_BO/Person.php (hunk 4 above)
Comment (josegus) [reply src/Faker/Provider/es_BO/Person.php]: Done

File: .idea/Faker.iml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pimjansen) [inline .idea/Faker.iml]: This file should not be in the repo

File: .idea/modules.xml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pimjansen) [inline .idea/modules.xml]: Same for this one

File: .idea/vcs.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pimjansen) [inline .idea/vcs.xml]: And this

File: .idea/workspace.xml [hunk 8]
```diff
@@ -0,0 +1,660 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pimjansen) [inline .idea/workspace.xml]: And all other `.idea` files

File: src/Faker/Provider/es_BO/Address.php [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+        '{{streetName}} # {{buildingNumber}} ',
+        '{{streetName}} # {{buildingNumber}} {{secondaryAddress}}',
+    );
+    protected static $addressFormats = array(
+        "{{streetAddress}}\n{{city}}, {{state}}",
+    );
+    protected static $secondaryAddressFormats = array('Dpto. ###', 'Hab. ###', 'Piso #', 'Piso ##');
+
+    /**
+     * @example ''
```
Comment (pimjansen) [inline src/Faker/Provider/es_BO/Address.php]: `@example` keywords are not really needed, please remove

**Comment by pimjansen** [comment]
@josegus bump, could you please review comments? Thanks a lot

File: src/Faker/Provider/es_BO/Address.php (hunk 9 above)
Comment (josegus) [reply src/Faker/Provider/es_BO/Address.php]: Ok, I will check it out as soon as possible

**Comment by pimjansen** [comment]
Im going to close this PR since it has been stale for another 14 days. Feel free to raise a new PR or reopen it.

Thanks!

**Pull request closed without merging**